.github/workflows/ci.yml

```diff
@@ -10,6 +10,9 @@ on:
 permissions:
   contents: read # to fetch code (actions/checkout)
 
+env:
+  NODE_VERSION: 18
```

Comment:
I think we can choose LTS version `lts/*`

Reply:
Thank you for your help, I have modified `NODE_VERSION` to `lts/*`.